django/db/backends/base/schema.py

```diff
@@ -990,14 +995,21 @@ def _index_include_sql(self, model, columns):
             columns=Columns(model._meta.db_table, columns, self.quote_name),
         )
 
-    def _create_index_sql(self, model, fields, *, name=None, suffix='', using='',
+    def _create_index_sql(self, model, *, fields=None, name=None, suffix='', using='',
```

Comment:
I've pushed changing `fields` to an optional kwarg-only to a separate PR, #13802.

Reply:
Merged and rebased.